store/bigtable/bigtable.go

```diff
@@ -269,9 +268,8 @@ func (s *Store) processWriteQueue(queue chan *mdata.ChunkWriteRequest, meter *st
 						failedMutations = append(failedMutations, muts[i])
 						retryBuf = append(retryBuf, buf[i])
 					} else {
-						buf[i].Metric.SyncChunkSaveState(buf[i].Chunk.Series.T0)
-						mdata.SendPersistMessage(buf[i].Key.String(), buf[i].Chunk.Series.T0)
-						log.Debugf("btStore: save complete. %s:%d %v", buf[i].Key, buf[i].Chunk.Series.T0, buf[i].Chunk)
+						buf[i].Callback()
```

Comment:
`Callback()` should be optional, so check if it is nil before trying to call it.